### PR TITLE
Add Bluetoothctl Wrapper

### PR DESCRIPTION
**Pull request by johnshamoon**

### CODE REVIEW

File: src/bluetoothctl.py [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+    pass
+
+
+class Bluetoothctl:
+    """
+    A wrapper for bluetoothctl.
+
+    Provides Bluetooth capabilities.
+    """
+
```
Comment (changnor) [inline src/bluetoothctl.py]: extra space on purpose?

Comment (johnshamoon) [reply src/bluetoothctl.py]: Yes.

File: docs/index.rst [hunk 2]
```diff
@@ -3,12 +3,13 @@ Welcome to Robotic Leader Follower's documentation!
 
 Robotic Leader Follower is a Wayne Sttate University senior capstone project to
```
Comment (zeinyoussef) [inline docs/index.rst]: We should probably fix this typo at some point but it doesn't matter too much to me for this PR.

File: src/bluetoothctl.py [hunk 3]
```diff
@@ -0,0 +1,336 @@
+"""
+bluetoothctl
```
[the remark this answers is not in the document]
Comment (johnshamoon) [reply src/bluetoothctl.py]: No. This is the module name.

File: src/bluetoothctl.py [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+import pexpect
+import subprocess
+import time
+
+
+class BluetoothctlError(Exception):
+    """
+    Bluetoothctl exception class.
+
+    The exception is raised when bluetoothctl fails to start.
```
[the remark this answers is not in the document]
Comment (johnshamoon) [reply src/bluetoothctl.py]: No. This is the linux command's name.

File: src/bluetoothctl.py [hunk 5]
```diff
@@ -0,0 +1,336 @@
...
+    """
+    pass
+
+
+class Bluetoothctl:
+    """
+    A wrapper for bluetoothctl.
```
[the remark this answers is not in the document]
Comment (johnshamoon) [reply src/bluetoothctl.py]: This is intentional.

**Pull request merged**